astropy/units/core.py

```diff
@@ -2173,7 +2174,7 @@ class Unit(NamedUnit, metaclass=_UnitMetaClass):
         canonical (short) name, and the rest of the elements are
         aliases.
 
-    represents : UnitBase instance
+    represents : UnitBase, str, or Quantity, optional
```

Comment:
UnitLike should be added to the glossary in the docs! Then it can be used here as |UnitLike| if you add in the shortcut in conf.py

Reply:
We actually already have `unit-like` in the glossary - of course, that spelling I cannot use for the typing, but I now use it here. Really an oversight not to have used it before!

Reply:
We should hook them all together! The type definition with the glossary term with the shortcut, etc.